frontend/src/config.ts

```diff
@@ -17,6 +18,9 @@ if (process.env.NODE_ENV === 'test') {
   gaInitializeOptions = {
     testMode: true,
   }
+
+  // Disable announcements in test environments
+  announcementActive = 'false'
```

Comment:
What do you think about disabling announcements for tests? I think it's fine since none of our tests rely on announcements currently.

Reply:
I think it's fine. Given that for most of the flows, that's not the main thing we're testing for.